frontend/javascripts/statusbar.js

```diff
@@ -0,0 +1,177 @@
+import Cube from "oxalis/model/bucket_data_handling/data_cube";
+import { V3 } from "libs/mjs";
+import Model from "oxalis/model";
+
+type OwnProps = {||};
+type StateProps = {|
+  activeResolution: Vector3,
+  activeViewport: OrthoView,
+  mousePosition: ?Vector2,
+|};
+type Props = {| ...OwnProps, ...StateProps |};
+type State = {||};
+
+const borderToggleButtonMargin = 40;
+const spaceBetweenItems = 20;
+
+const statusbarStyle: Object = {
+  paddingTop: 2,
+  marginLeft: borderToggleButtonMargin,
+  marginRight: borderToggleButtonMargin,
+};
+
+const hasSegmentation = () => Model.getSegmentationLayer() != null;
+
+class Statusbar extends React.PureComponent<Props, State> {
+  getSegmentationCube(): Cube {
+    const segmentationLayer = Model.getSegmentationLayer();
+    return segmentationLayer.cube;
+  }
+
+  getPosString(pos: Vector3) {
+    return V3.floor(pos).join(",");
+  }
+
+  getZoomShortcut() {
+    return (
+      <span key="zoom">
+        <span
+          key="zoom-i"
+          className="keyboard-key-icon-small"
+          style={{
+            borderColor: "rgba(255, 255, 255, 0.67)",
+          }}
+        >
+          I
+        </span>{" "}
+        /{" "}
+        <span
+          key="zoom-o"
+          className="keyboard-key-icon-small"
+          style={{
+            borderColor: "rgba(255, 255, 255, 0.67)",
+          }}
+        >
+          O
+        </span>{" "}
+        Zoom in/out
+      </span>
+    );
+  }
+
+  getShortcuts() {
+    const { activeViewport } = this.props;
+    return (
+      <Space size={spaceBetweenItems} style={{ textAlign: "right" }}>
+        <span>
+          <img
+            className="keyboard-mouse-icon"
+            src="/assets/images/icon-mouse-left-drag.svg"
+            alt="Mouse Left Drag"
+            style={{ height: 12 }}
+          />
+          Move
+        </span>
+        <span>
+          <img
+            className="keyboard-mouse-icon"
+            src="/assets/images/icon-mousewheel.svg"
+            alt="Mouse Wheel"
+            style={{ height: 12 }}
+          />
+          Move along 3rd axis
+        </span>
+        {this.getZoomShortcut()}
+        {activeViewport === OrthoViews.TDView && (
+          <span>
+            <img
+              className="keyboard-mouse-icon"
+              src="/assets/images/icon-mouse-right.svg"
+              alt="Mouse Right"
+              style={{ height: 12 }}
+            />
+            Rotate 3D View
+          </span>
+        )}
+      </Space>
+    );
+  }
+
+  getCellInfo(globalMousePosition: ?Vector3) {
+    const segmentationLayerName = Model.getSegmentationLayer().name;
+    const cube = this.getSegmentationCube();
+    const renderedZoomStepForMousePosition = api.data.getRenderedZoomStepAtPosition(
+      segmentationLayerName,
+      globalMousePosition,
+    );
+    const getIdForPos = (pos, usableZoomStep) =>
+      pos && cube.getDataValue(pos, null, usableZoomStep);
```

Comment:
In the "Volume" tab (aka mapping-info-tab) one can select/enable a mapping which changes the segment ids according to a JSON (or hdf5) "dictionary". I'd argue that we should show the mapped segment id in that case here. This should be possible via something like this:

```
id = cube.getDataValue(...)
return cube.mapId(id)
```

there's no need to check whether the mapping is enabled, since `mapId` will do so automatically.

Additionally, I would change the appearance of the id so that it shows "Cell: 123 (mapped)" *iff* a mapping is enabled. This should probably happen outside of the `getIdForPos` function, since the result wouldn't be an id anymore (but a string). You can check whether a mapping is enabled via `cube.isMappingEnabled()`.